improver_tests/developer_tools/test_MOMetadataInterpreter.py

```diff
@@ -0,0 +1,401 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2021 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Unit tests for the MOMetadataInterpreter plugin"""
+
+import numpy as np
+import pytest
+from iris.coords import CellMethod
+
+from improver_tests.developer_tools import ensemble_fixture  # ensemble_cube
+from improver_tests.developer_tools import interpreter_fixture  # interpreter
+from improver_tests.developer_tools import landmask_fixture  # landmask_cube
+from improver_tests.developer_tools import (
+    percentile_fixture,  # wind_gust_percentile_cube,
+)
+from improver_tests.developer_tools import (
+    probability_above_fixture,  # probability_above_cube,
+)
+from improver_tests.developer_tools import (
+    probability_below_fixture,  # blended_probability_below_cube,
+)
+from improver_tests.developer_tools import snow_level_fixture  # snow_level_cube
+from improver_tests.developer_tools import spot_fixture  # blended_spot_median_cube
+from improver_tests.developer_tools import wind_direction_fixture  # wind_direction_cube
+from improver_tests.developer_tools import wxcode_fixture  # wxcode_cube
```

Comment:
I now understand what is meant by "lawful evil".  I will fix this and put the comments / translation elsewhere!